uber/automated_emails.py

```diff
@@ -385,26 +393,30 @@ def __init__(self, conf):
 
 MarketplaceEmail('Your {EVENT_NAME} Dealer registration has been approved', 'dealers/approved.html',
                  lambda g: g.status == c.APPROVED,
-                 needs_approval=False)
+                 needs_approval=False,
+                 ident='dealer_reg_approved')
 
 MarketplaceEmail('Reminder to pay for your {EVENT_NAME} Dealer registration', 'dealers/payment_reminder.txt',
                  lambda g: g.status == c.APPROVED and days_after(30, g.approved)() and g.is_unpaid,
-                 needs_approval=False)
+                 needs_approval=False,
+                 ident='dealer_reg_payment_reminder')
 
 MarketplaceEmail('Your {EVENT_NAME} Dealer registration is due in one week', 'dealers/payment_reminder.txt',
                  lambda g: g.status == c.APPROVED and g.is_unpaid,
                  when=days_before(7, c.DEALER_PAYMENT_DUE, 2),
-                 needs_approval=False)
+                 needs_approval=False,
+                 ident='dealer_reg_payment_reminder_due_soon')
 
 MarketplaceEmail('Last chance to pay for your {EVENT_NAME} Dealer registration', 'dealers/payment_reminder.txt',
                  lambda g: g.status == c.APPROVED and g.is_unpaid,
                  when=days_before(2, c.DEALER_PAYMENT_DUE),
-                 needs_approval=False)
+                 needs_approval=False,
+                 ident='dealer_reg_payment_reminder_last_chance')
 
 MarketplaceEmail('{EVENT_NAME} Dealer waitlist has been exhausted', 'dealers/waitlist_closing.txt',
                  lambda g: g.status == c.WAITLISTED,
                  when=days_after(0, c.DEALER_WAITLIST_CLOSED),
-                 ident='uber_marketplace_waitlist_exhausted')
+                 ident='dealer_waitlist_exhausted')
 
```

Comment:
This was the only pre-existing `ident` that I had to change to fit with our more terse naming scheme.